product-service/serverless.yml

```diff
@@ -9,8 +9,17 @@ provider:
   stage: dev
   region: eu-west-1
 
+# you can define service wide environment variables here
+  environment:
```

Comment:
You can hide sensitive data in the .env file
https://www.serverless.com/framework/docs/providers/aws/guide/variables/#referencing-environment-variables

Reply:
Please do not commit your credentials - it's very unsecure, you should understand it
At least use dotenv package, as Anton said

Reply:
I had some issues with git, here is main branch without public credentials and .serverless https://github.com/musiichuk/aws-shop-be/tree/main/product-service take a look and change the score @wwrcth